src/instructlab/configuration.py

```diff
@@ -266,6 +266,7 @@ class _serve_vllm(BaseModel):
 
     llm_family: str = ""
     max_startup_attempts: int | None = None
+    gpus: Optional[int] = None
```

Comment:
We should probably change the evaluate default to None as well.  It is currently set to default to 1, so it will override serves config by default.